edx_solutions_api_integration/courses/views.py

```diff
@@ -369,6 +369,7 @@ def get(self, request, course_id, content_id=None):
         data_blocks = get_blocks(
             request,
             usage_key,
+            user=request.user,
```

Comment:
I'm not sure about this change ^ I suspect it's more or less pointless since it's only used in a server-to-server API  (i.e. with `AnonymousUser`) and is not part of the regular flow for viewing courses in Apros. Can you please leave this line out for now? (Unless you know some reason to leave it in).

Reply:
@bradenmacdonald no reason. Removed.